web-jehlomat/src/screens/Nalezy/Components/FilterByRange.tsx

```diff
@@ -1,25 +1,48 @@
-import React, { ChangeEvent, FunctionComponent, useEffect, useState } from 'react';
-import dayjs, { Dayjs } from 'dayjs';
+import React, { ChangeEvent, FunctionComponent, useCallback, useEffect, useState } from 'react';
+import { useSetRecoilState } from 'recoil';
+import dayjs from 'dayjs';
+import { RangeKind } from 'screens/Nalezy/types/RangeKind';
+import { Range as DateRange } from 'screens/Nalezy/types/Range';
+import { Filtering } from 'screens/Nalezy/types/Filtering';
+import { filteringState } from 'screens/Nalezy/store';
 import { Filter, Range } from './Filter';
-import Select from './Select';
 import { DatePicker } from './Input';
-import { RangeKind } from '../types/RangeKind';
-
-interface FilterPeriodProps {
-    onFilter: (kind: RangeKind, from: Dayjs | null, to: Dayjs | null) => void;
-    onReset: () => void;
-}
+import Select from './Select';
 
-const FilterByRange: FunctionComponent<FilterPeriodProps> = ({ onFilter, onReset }) => {
+const FilterByRange: FunctionComponent = () => {
     const [kind, setKind] = useState<RangeKind | ''>('');
```

Comment:
You can just use it like `useState<RangeKind'>` and state is undefined by default after that. I think it's cleaner to use it like this, then setting empty string, especially in places where final value type is diffent. What do you think?

Reply:
Nope, I need to be to reset select to an empty value, with undefined you can't.